packages/variable-editor/src/components/variables/detail/Value.tsx

```diff
@@ -34,6 +34,8 @@ export const Value = ({ variable, onChange }: ValueProps) => {
             onValueChange={(value: string) => onChange([{ key: 'value', value: value }])}
           />
         );
+      case 'file':
+        return <Message message='Value is served by a file.' variant='info' />;
```

Comment:
I would prefer if there was still an empty, disabled input field.
I know it would serve absolutely no functional purpose, but I think the UI would look better and be more consistent.

Reply:
Fine for me 👍